docs/docs/fam/fam_doc.md

```diff
@@ -0,0 +1,85 @@
+# ARLAS File and Archive Management (FAM)
+
+FAM is ARLAS File and Archive Management service. The endpoint lists files in a directory and can list contained archives.
+
+
+For more details, see the [FAM API documentation](fam_api.md)
+
+## Using FAM
+
+In the following examples, we will:
+
+- list files in `DIMAP`
+- list archives in `DIMAP`
+
+!!! note
+    The service is deployed on `ARLAS_ENDPOINT` 
+
+### FAM Configuration
```

Comment:
this FAM Configuration should go to the project readme, not the doc that goes to docs.arlas.io